### ISSUE TITLE
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'sproutemail_notificationemails.viewContext' in 'field list'

### ISSUE THREAD

**Issue by outline4** [barrelstrength/craft-sprout-email#128]
### Description (screenshots, logs, ...)

When I try to access Sprout Email I get this error Message: 

The SQL being executed was: SELECT EXISTS(SELECT `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`slug`, `elements_sites`.`siteId`, `elements_sites`.`enabled` AS `enabledForSite`, `elements_sites`.`uri`, `sproutemail_notificationemails`.`viewContext`, `sproutemail_notificationemails`.`titleFormat`, `sproutemail_notificationemails`.`sendRule`, `sproutemail_notificationemails`.`emailTemplateId`, `sproutemail_notificationemails`.`eventId`, `sproutemail_notificationemails`.`settings`, `sproutemail_notificationemails`.`subjectLine`, `sproutemail_notificationemails`.`defaultBody`, `sproutemail_notificationemails`.`recipients`, `sproutemail_notificationemails`.`cc`, `sproutemail_notificationemails`.`bcc`, `sproutemail_notificationemails`.`listSettings`, `sproutemail_notificationemails`.`fromName`, `sproutemail_notificationemails`.`fromEmail`, `sproutemail_notificationemails`.`replyToEmail`, `sproutemail_notificationemails`.`singleEmail`, `sproutemail_notificationemails`.`enableFileAttachments`, `sproutemail_notificationemails`.`dateCreated`, `sproutemail_notificationemails`.`dateUpdated`, `sproutemail_notificationemails`.`fieldLayoutId`, `content`.`id` AS `contentId`, `content`.`title`, `content`.`field_block_switch`, `content`.`field_space`, `content`.`field_cf_account_intro_richText`, `content`.`field_adresse`, `content`.`field_cf_rennkontakte_adresse`, `content`.`field_cf_dates_preparse`, `content`.`field_cf_altText`, `content`.`field_cf_freeform`, `content`.`field_anrede`, `content`.`field_align`, `content`.`field_autokennzeichen`, `content`.`field_bank`, `content`.`field_cf_pinnwand_beitrag`, `content`.`field_cf_userRights_checkboxes`, `content`.`field_cf_memberDesc_richText`, `content`.`field_cf_shop_cat_richText`, `content`.`field_contentWidth`, `content`.`field_code`, `content`.`field_color`, `content`.`field_adWizard_details`, `content`.`field_embedCode`, `content`.`field_cf_end_date`, `content`.`field_cf_weitereRennen_fahrzeuge`, `content`.`field_cf_shop_cat_color`, `content`.`field_cf_colorScheme`, `content`.`field_fax`, `content`.`field_firma`, `content`.`field_fixnetz`, `content`.`field_cf_footer_address_richText`, `content`.`field_cf_footer_contact`, `content`.`field_cf_member_funktion`, `content`.`field_geburtsdatum`, `content`.`field_columnSize`, `content`.`field_handy`, `content`.`field_backgroundColor`, `content`.`field_iban`, `content`.`field_cf_blog_id`, `content`.`field_cf_rennkontakte_id`, `content`.`field_isMainTitle_switch`, `content`.`field_cf_no_permission_richText`, `content`.`field_kleidergroesse`, `content`.`field_cf_weitereRennen_kontakt`, `content`.`field_laendercode`, `content`.`field_layoutType`, `content`.`field_imageCaption`, `content`.`field_cf_link`, `content`.`field_linkUrl`, `content`.`field_lizenznummer`, `content`.`field_cf_metaDescription`, `content`.`field_cf_metaKeywords`, `content`.`field_cf_metaTitle`, `content`.`field_withSiteChooser`, `content`.`field_withSubnav`, `content`.`field_cf_rennen_ort`, `content`.`field_cf_metaPlacename`, `content`.`field_plzOrt`, `content`.`field_cf_metaPosition`, `content`.`field_cf_pages_quicklinkTitle`, `content`.`field_cf_ranking_tableMaker`, `content`.`field_ratio`, `content`.`field_cf_metaRegion`, `content`.`field_cf_rubrik_color`, `content`.`field_rufname`, `content`.`field_specialLink`, `content`.`field_cf_stand_date`, `content`.`field_cf_start_date`, `content`.`field_cf_subnav`, `content`.`field_table`, `content`.`field_cf_blog_teaserText`, `content`.`field_cf_weitereRennen_teilnehmer`, `content`.`field_cf_rennkontakte_telefon`, `content`.`field_template`, `content`.`field_richText`, `content`.`field_text`, `content`.`field_fileTitle`, `content`.`field_titel`, `content`.`field_titleFormat`, `content`.`field_cf_titleTrail`, `content`.`field_cf_rangliste_subtitle`, `content`.`field_subTitle`, `content`.`field_userId`, `content`.`field_cf_rennen_veranstaltung`, `content`.`field_cf_rennkontakte_veranstaltung`, `content`.`field_cf_hideNav_switch`, `content`.`field_video`, `content`.`field_cf_member_firstName`, `content`.`field_cf_rennen_webseite`, `content`.`field_cf_rennkontakte_webseite`, `content`.`field_cf_website`, `content`.`field_webseite`, `elements`.`draftId`, `drafts`.`sourceId` AS `draftSourceId`, `drafts`.`creatorId` AS `draftCreatorId`, `drafts`.`name` AS `draftName`, `drafts`.`notes` AS `draftNotes`
FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`
FROM `craft_elements` `elements`
INNER JOIN `craft_sproutemail_notificationemails` `sproutemail_notificationemails` ON `sproutemail_notificationemails`.`id` = `elements`.`id`
INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`
INNER JOIN `craft_content` `content` ON (`content`.`elementId` = `elements`.`id`) AND (`content`.`siteId` = `elements_sites`.`siteId`)
INNER JOIN `craft_drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
WHERE (`elements_sites`.`siteId`='1') AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`drafts`.`sourceId` IS NULL) AND (`elements`.`revisionId` IS NULL)
ORDER BY `elements`.`dateCreated` DESC) `subquery`
INNER JOIN `craft_sproutemail_notificationemails` `sproutemail_notificationemails` ON `sproutemail_notificationemails`.`id` = `subquery`.`elementsId`
INNER JOIN `craft_elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`
INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`
INNER JOIN `craft_content` `content` ON `content`.`id` = `subquery`.`contentId`
INNER JOIN `craft_drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
ORDER BY `elements`.`dateCreated` DESC)
Error Info: Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'sproutemail_notificationemails.viewContext' in 'field list'
)
↵
Caused by: PDOException

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'sproutemail_notificationemails.viewContext' in 'field list'

in /home/outline4/www/motorsport.in-arbeit.ch/vendor/yiisoft/yii2/db/Command.php at line 1290

### Steps to reproduce

1. I've installed the plugin from the plugin store. 
2. After installing, I saw that the plugin was deactivated. 
3. I've activated it. 
4. I've tried to access the plugin in the sidebar and the only thing I see is this big fat ugly error message :) 

### Additional info

- Craft version: 3.3.12
- PHP version: 7.2.22
- Database driver & version:  MySQL 5.5.5
- Plugins & versions: Sprout Email: 4.0.5

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_12pvn)

**Comment by BenParizek** [comment]
Thanks @outline4. We're looking into a fix. Will update here once we get it out.

**Comment by andrelopez** [comment]
Hi @outline4 Could you please uninstall the plugin and then remove the following table manually from your database? `sproutemail_notificationemails` 

Then install the plugin again?

It's possible that you installed the plugin in the past and that table remains in the database with a missing column.

**Comment by outline4** [comment]
Hi, I've deinstalled the plugin and then removed it.

Then I've removed the following tables:  
craft_sproutemail_notificationemails and craft_sproutemail_sentemail

Btw: There were still tables from "sprout fields" (that I didn't install) that I've also deleted and some tables from "sprout reports" (that I also didn't have installed). 

Then I've installed sprout email. It worked, but it was deactivated. 
After activating it, it seems to work! 

Yay!
